lib/routemap.c

```diff
@@ -3101,27 +3101,24 @@ static void clear_route_map_helper(struct route_map *map)
 		index->applied_clear = index->applied;
 }
 
-DEFUN (rmap_clear_counters,
```

Comment:
Why do we need both commits?  Let's just cut to the chase and have this one?

Reply:
Just to maintain two distinct authorship, but I have combined into one now. :)